ckanext/dhis2harvester/templates/source/pivot_table_new_3.html

```diff
@@ -60,51 +60,63 @@ <h2>Configure Columns and Categories</h2>
-                  id='target_column;{};{}'.format(pivot_table.id, column.value),
-                  label='column',
-                  options=column_options,
-                  selected=column_data.target_column if column_data else column_options[0],
-                  error=errors.pivot_table_id,
-                  classes=['control-medium', 'col-xs-12', 'col-md-4']
-                  )
-              }}
-
-              {% for category, values in data.column_config[target_type].categories.iteritems() %}
-                {% set category_options = [] %}
-                {% for value in values %}
-                  {% do category_options.append({'value': value, 'text': value}) %}
+                {% set column_options = [] %}
+                {% for column in data.column_config[target_type].columns %}
+                  {% do column_options.append({'value': column, 'text': column}) %}
                 {% endfor %}
                 {{ form.select(
-                    'category;{};{};{}'.format(pivot_table.id, category, column.value),
-                    id='category;{};{};{}'.format(pivot_table.id, category, column.value),
-                    label=category,
-                    options=category_options,
-                    selected=column_data.categories[category] if column_data else category_options[0].value,
+                    'target_column;{};{}'.format(pivot_table.id, column.value),
+                    id='target_column;{};{}'.format(pivot_table.id, column.value),
+                    label='column',
+                    options=column_options,
+                    selected=column_data.target_column if column_data else column_options[0],
                     error=errors.pivot_table_id,
                     classes=['control-medium', 'col-xs-12', 'col-md-4']
                     )
                 }}
-              {% endfor %}
-            {% endif %}
+
+                {% for category, values in data.column_config[target_type].categories.iteritems() %}
+                  {% set category_options = [] %}
+                  {% for value in values %}
+                    {% do category_options.append({'value': value, 'text': value}) %}
+                  {% endfor %}
+                  {{ form.select(
+                      'category;{};{};{}'.format(pivot_table.id, category, column.value),
+                      id='category;{};{};{}'.format(pivot_table.id, category, column.value),
+                      label=category,
+                      options=category_options,
+                      selected=column_data.categories[category] if column_data else category_options[0].value,
+                      error=errors.pivot_table_id,
+                      classes=['control-medium', 'col-xs-12', 'col-md-4']
+                      )
+                  }}
+                {% endfor %}
+              {% endif %}
+            </div>
+            <div class="row">
+              {{ form.checkbox(
```

Comment:
Can we just wrap the check box in a bootstrap col  (keep it as a child of "row")
```suggestion
              <div class="col-xs-12">
                  {{ form.checkbox(
```

Reply:
The aim is to get the check box aligned with the other elements.